deployment/helm/image-metadata-orchestrator-base/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "image-metadata-orchestrator-base.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "image-metadata-orchestrator-base.fullname" -}}
```

Comment:
We have the "image metadata collector".
This app name could be "cluster-image-scanner-orchestrator"

Reply:
We agreed to keep the name as is.